Filtered_kernel/include/CGAL/Kernel_profiler.h

```diff
@@ -43,77 +43,13 @@ struct Primitive_profiler
     Primitive_profiler(const P& p = P())
       : P(p) {}
 
-    template <class A1>
+    template <class ... A>
     result_type
-    operator()(const A1 &a1) const
+    operator()(const A&& ... a) const
     {
 	CGAL_KERNEL_PROFILER;
-	return P::operator()(a1);
+	return P::operator()(std::forward(a)...);
```

Comment:
Fix the perfect forwarding:
```suggestion
	return P::operator()(std::forward<A>(a)...);
```